hive.go

```diff
@@ -33,6 +33,8 @@ func main() {
 		simTestLimit    = flag.Int("sim.testlimit", 0, "Max `number` of tests to execute per client (interpreted by simulators).")
 		simTimeLimit    = flag.Duration("sim.timelimit", 0, "Simulation `timeout`. Hive aborts the simulator if it exceeds this time.")
 		simLogLevel     = flag.Int("sim.loglevel", 3, "Selects log `level` of client instances. Supports values 0-5.")
+		simDevMode	= flag.Bool("sim-dev", false, "Only starts the simulator API endpoint (listening at 127.0.0.1:3000 by default) without starting any simulators.")
+		simDevModeAPIPort   = flag.Uint("sim-dev.port", 3000, "override the default port which the simulator API listens on when --sim-dev is provided")
```

Comment:
Flag should be `--dev.addr <string>`, i.e. allow passing full listening address.